Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -69,5 +74,29 @@ def commit(self):
             distances = inv(distances)
         self.send("Distances", distances)
 
+    def send_report(self):
+        items = OrderedDict()
+        if self.normalization_method == 0:
+            if self.inversion_method == 0:
+                items['Transformation'] = 'None'
+            else:
+                items['Transformation'] = 'Inversion (' + \
+                    self.inversion_options[self.inversion_method][0] + ')'
+        else:
+            if self.inversion_method == 0:
+                items['Transformation'] = ('Normalization {}'.
+                                            format(self.normalization_options
+                                            [self.normalization_method][0])
+                                            .lower())
```

Comment:
I don't mind changing to what you said. Just that it would require 2 extra variables and now also the code is easily understandable although not that much beautiful.
You say, I follow :smile_cat: